         
int64 vagon_type
string seat_sequence_type
int64 index
---
#result
string result
